build_product_images.py

```diff
@@ -60,6 +60,11 @@ def parse_args():
         required=True,
         type=check_architecture_input
     )
+    parser.add_argument(
+        "--path",
```

Comment:
According to the docker docs, this is the _organization namespace_ [1] so I would call it `--organization` and also add the short option `-o`

[1] https://docs.docker.com/glossary/#organization-name

Reply:
Done